test/duration.mjs

```diff
@@ -371,7 +371,9 @@ describe('Duration', () => {
     });
     it('unrepresentable number is not allowed', () => {
       units.forEach((unit, ix) => {
+        // eslint-disable-next-line @typescript-eslint/no-loss-of-precision,no-loss-of-precision
```

Comment:
After some experimentation, I found that I needed to disable both variations of this rule because the prettier command-line tool would complain if one of them wasn't disabled, while the in-IDE prettier would complain if the other one wasn't disabled.  There's probably a better way to handle this (maybe exclude code files from running using the prettier command line and rely on ESLint only for code files, using prettier only for non-code files) but I was lazy and didn't want to spend the time to figure that out given that the workaround above is only needed in a few spots.